tests/packaging_util_tests.py

```diff
@@ -145,6 +146,23 @@ def test_build_skill_spec(self):
         self.assertEqual(valid_spec, skill_spec)
 
     # TODO: Actually validate exception cases? DM
+        
+    def test_install_packages_from_pip(self):
+        import pip
+        from neon_utils.packaging_utils import install_packages_from_pip
+
+        with patch.object(pip, 'main', return_value=0) as mock_method:
+            test_result = install_packages_from_pip("neon-audio", ["pip-install-test"])
+        
+            self.assertEqual(0, test_result)
+            mock_method.assert_called_once()
+
+            mock_method.reset_mock()
+            test_result = install_packages_from_pip("neon-audio", ["pip-install-test", "pip-install-another"])
+        
+            self.assertEqual(0, test_result)
+            mock_method.assert_called_once()
```

Comment:
Can you check the contents of the requirements file in one of the test cases here?